lib/screens/explore_screens/regions_screens/regions_main/components/region_event_card/interactor/viewmodels/region_event_card_bloc.dart

```diff
@@ -8,10 +13,25 @@ part 'region_event_card_event.dart';
 part 'region_event_card_state.dart';
 
 class RegionEventCardBloc extends Bloc<RegionEventCardEvent, RegionEventCardState> {
-  RegionEventCardBloc() : super(RegionEventCardState.initial()) {
+  late StreamSubscription<List<RegionEventModel>> _eventListener;
+
+  RegionEventCardBloc(RegionEventModel event) : super(RegionEventCardState.initial()) {
+    _eventListener = FirebaseDatabaseRegionsRepository().getEventsForRegion(event.regionAccount).listen((events) {
+      final found = events.singleWhereOrNull((i) => i.id == event.id);
+      if (found != null) {
+        add(OnLoadRegionEventMembers(found.users));
+      }
+    });
```

Comment:
@gguijarro-c-chwy I tried to create my own call to firebase for a document but it didn't work for me I didn't listen to the changes at the end I used the one that listens to all the regions and I only check for the event of that card.